### DISCUSSION TITLE
Dashboard on Autostart ?

### DISCUSSION THREAD

**Discussion by DennisCor** [gfunkbus76/Digifiz-Dash#7] [Q&A]
Hi Gavin,

Did you manage to start the pygame program (main.py) after boot your raspberry ?
Doesn't seem to be that easy.

Cheers Dennis

**Comment by gfunkbus76** [comment]
Hi Dennis!!

I'm sorry for your struggles! Unfortunately, I haven't had time of late to do any further development or testing as such. I'm busy working on the mechanical side of the project to get my bus running... Then I'll have to bounce back here to develop more on the software side.

I was imagining creating a bash file in the autostart lxde folder/file. To auto run the Python app. As mentioned, I am haven't got that far myself. Maybe others have?

Will try and rig up my system soon for more testing. Sorry you are one of the Guinea pigs...
